tests/test_homo_sapiens.py

```diff
@@ -26,33 +26,42 @@ def test_str(self):
-        self.assertEqual(genome.chromosomes["chr10"].length, 133797422)
-        self.assertEqual(genome.chromosomes["chr11"].length, 135086622)
-        self.assertEqual(genome.chromosomes["chr12"].length, 133275309)
-        self.assertEqual(genome.chromosomes["chr13"].length, 114364328)
-        self.assertEqual(genome.chromosomes["chr14"].length, 107043718)
-        self.assertEqual(genome.chromosomes["chr15"].length, 101991189)
-        self.assertEqual(genome.chromosomes["chr16"].length, 90338345)
-        self.assertEqual(genome.chromosomes["chr17"].length, 83257441)
-        self.assertEqual(genome.chromosomes["chr18"].length, 80373285)
-        self.assertEqual(genome.chromosomes["chr19"].length, 58617616)
-        self.assertEqual(genome.chromosomes["chr20"].length, 64444167)
-        self.assertEqual(genome.chromosomes["chr21"].length, 46709983)
-        self.assertEqual(genome.chromosomes["chr22"].length, 50818468)
-        self.assertEqual(genome.chromosomes["chrX"].length, 156040895)
+        self.assertEqual(genome.chromosomes["chr1"].length, 249218992)
+        self.assertEqual(genome.chromosomes["chr2"].length, 243090997)
+        self.assertEqual(genome.chromosomes["chr3"].length, 197874528)
+        self.assertEqual(genome.chromosomes["chr4"].length, 191028645)
+        self.assertEqual(genome.chromosomes["chr5"].length, 180715810)
+        self.assertEqual(genome.chromosomes["chr6"].length, 171051316)
+        self.assertEqual(genome.chromosomes["chr7"].length, 159126774)
+        self.assertEqual(genome.chromosomes["chr8"].length, 146301156)
+        self.assertEqual(genome.chromosomes["chr9"].length, 141109979)
+        self.assertEqual(genome.chromosomes["chr10"].length, 135503693)
+        self.assertEqual(genome.chromosomes["chr11"].length, 134945522)
+        self.assertEqual(genome.chromosomes["chr12"].length, 133778796)
+        self.assertEqual(genome.chromosomes["chr13"].length, 115106996)
+        self.assertEqual(genome.chromosomes["chr14"].length, 107288377)
+        self.assertEqual(genome.chromosomes["chr15"].length, 102506108)
+        self.assertEqual(genome.chromosomes["chr16"].length, 90163588)
+        self.assertEqual(genome.chromosomes["chr17"].length, 81045269)
+        self.assertEqual(genome.chromosomes["chr18"].length, 78015180)
+        self.assertEqual(genome.chromosomes["chr19"].length, 59097160)
+        self.assertEqual(genome.chromosomes["chr20"].length, 62949445)
+        self.assertEqual(genome.chromosomes["chr21"].length, 48100155)
+        self.assertEqual(genome.chromosomes["chr22"].length, 51229805)
+        self.assertEqual(genome.chromosomes["chrX"].length, 154929412)
         self.assertEqual(genome.chromosomes["chrY"].length, 57227415)
 
+    def test_mean_recombination_rate(self):
+        genome = homo_sapiens.genome
+        highest_rr = 0
+        lowest_rr = 10000
+        for chrom, chrom0 in genome.chromosomes.items():
```

Comment:
What is this testing?

Reply:
It just tests that the mean lies with the lowest and greatest value. It was the first sanity check I could think of.

open to suggestions for a better test.

Reply:
Recompute the mean using numpy and see if it's close?

Reply:
will do.

Reply:
I've got the code for this but I'm going to hold off on pushing it until #33 is pulled so I can iterate through chromosomes and get an `msprime` recombination_map for each one.